core-common/src/main/java/eu/arrowhead/common/database/entity/ChoreographerExecutorServiceDefinitionConnection.java

```diff
@@ -0,0 +1,85 @@
+package eu.arrowhead.common.database.entity;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import java.time.ZonedDateTime;
+
+@Entity
+@Table
+public class ChoreographerExecutorServiceDefinitionConnection {
```

Comment:
new table and entity: ChoreographerExecutorServiceDefinitionPrecondition

- id
- ChoreographerExecutorServiceDefinitionConnection.id
- ChoreographerExecutorServiceDefinition.id